process-engine/bpmn-studio
# ✨ Sign Windows build

## Changes

1. Sign the Windows Installer and the unpacked BPMN Studio.exe

## Issues

Closes #1577 

PR: #2108 

## How to test the changes

- download, install and start [this release](https://github.com/process-engine/bpmn-studio/releases/tag/v6.5.0-feature-4af929-kig3v1zv)

- update to [this signed release](https://github.com/process-engine/bpmn-studio/releases/tag/v6.5.0-feature-8e7a45-kihd4w6m) through the BPMN Studio update notification.

- to verify that the new BPMN Studio is signed, goto the installation directory and right click on `BPMN Studio.exe`, click on `Properties` -> `Digital Signature` like on the screen below:

![Bildschirmfoto 2020-12-09 um 14 58 07](https://user-images.githubusercontent.com/17065920/101639127-3448eb00-3a2f-11eb-8fd1-9a8a5536b52c.png)
![Bildschirmfoto 2020-12-09 um 14 58 36](https://user-images.githubusercontent.com/17065920/101639164-3e6ae980-3a2f-11eb-864c-de3374c47b80.png)


The two releases will be deleted after merging this PR.
